api/docs/tough-cookie.domainmatch.md

```diff
@@ -0,0 +1,110 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [tough-cookie](./tough-cookie.md) &gt; [domainMatch](./tough-cookie.domainmatch.md)
+
+## domainMatch() function
+
+Answers "does this real domain match the domain in a cookie?". The `domain` is the "current" domain name and the `cookieDomain` is the "cookie" domain name. Matches according to [RFC6265 - Section 5.1.3](https://www.rfc-editor.org/rfc/rfc6265.html#section-5.1.3)<!-- -->, but it helps to think of it as a "suffix match".
+
+**Signature:**
+
+```typescript
+export declare function domainMatch(domain?: Nullable<string>, cookieDomain?: Nullable<string>, canonicalize?: boolean): boolean | undefined;
+```
+
+## Parameters
+
+<table><thead><tr><th>
+
+Parameter
+
+
+</th><th>
+
+Type
+
+
+</th><th>
+
+Description
+
+
+</th></tr></thead>
+<tbody><tr><td>
+
+domain
+
+
+</td><td>
+
+Nullable&lt;string&gt;
+
+
+</td><td>
+
+_(Optional)_ The domain string to test
+
+
+</td></tr>
+<tr><td>
+
+cookieDomain
+
+
+</td><td>
+
+Nullable&lt;string&gt;
+
+
+</td><td>
+
+_(Optional)_ The cookie domain string to match against
+
+
+</td></tr>
+<tr><td>
+
+canonicalize
+
+
+</td><td>
+
+boolean
+
+
+</td><td>
+
+_(Optional)_ The canonicalize parameter toggles whether the domain parameters get normalized with canonicalDomain or not
+
+
+</td></tr>
+</tbody></table>
+**Returns:**
+
+boolean \| undefined
+
+## Remarks
+
+\#\#\# 5.1.3. Domain Matching
```

Comment:
Not rendered correctly 🫤

Reply:
I'll check to see if this is an existing issue with `@microsoft/api-documenter` and file one if not.  These do render correctly in WebStorm's doc tooltips as you can see below:

<img width="572" alt="Screenshot 2024-04-12 at 2 38 58 PM" src="https://github.com/salesforce/tough-cookie/assets/19131/1a443985-d1c3-4ac0-9532-4d0a804e4c83">